test/built-ins/Proxy/get/trap-is-undefined-receiver.js

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2016 Aleksey Shvayka. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 9.5.8
+description: >
+    [[Get]] (P, Receiver)
+
+    7. If trap is undefined, then
+        a. Return ? target.[[Get]](P, Receiver).
+---*/
+
+var target = {
+    get attr() {
+        return this;
+    }
+};
+
+var p = new Proxy(target, { get: undefined });
+assert.sameValue(p.attr, p);
+
+var pHerit = new Proxy(Object.create(target), { get: null });
+assert.sameValue(pHerit.attr, pHerit);
```

Comment:
how is this assertion different from the first one? Seems like it would be more interesting to make pHerit inherit `p`, and assert that since the handler doesn't have a "get" trap, it calls `target`'s accessor with `pHerit` as the receiver.

```js
var p = new Proxy(target, { get: undefined });

var pHerit = Object.create(p);
assert.sameValue(pHerit.attr, pHerit);
```